utils/tsdev.sh

```diff
@@ -11,9 +11,13 @@ help(){
 	echo "  -h, --help        Display this help message"
 	echo
 	echo "Commands:"
+	echo "  activate-history  Enable bash history persistence in the container"
 	echo "  build-api-cli     Build and install API and CLI clients"
 	echo "  celery            Start a celery worker"
 	echo "  logs              Follow docker container logs "
+	echo "  psql              Connect to the Timesketch PostgreSQL database in the dev container"
+	echo "  rebuild-dev       Stop, remove, and rebuild the dev environment (docker-compose down && docker-compose up --build -d)"
```

Comment:
Should we still call it `rebuild-dev` if we now not rebuild anymore? Would `reset-dev` be more appropriate? Also update the command here.

Reply:
good point